packages/jsii-dotnet-analyzers/NuGet.Metadata.props.t.js

```diff
@@ -2,17 +2,25 @@ const package = require('./package.json');
 
 process.stdout.write(`<Project>
   <PropertyGroup>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <IncludeSymbols>True</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <IncludeSource>True</IncludeSource>
-    <PackageOutputPath>..\\..\\bin\\$(Configuration)\\NuGet\\</PackageOutputPath>
     <PackageVersion>$(JsiiVersion)</PackageVersion>
     <Description>${package.description}</Description>
-    <ProjectUrl>${package.homepage}</ProjectUrl>
+    <PackageLicenseExpression>${package.license}</PackageLicenseExpression>
     <LicenseUrl>https://spdx.org/licenses/${package.license}.html</LicenseUrl>
+
+    <ProjectUrl>${package.homepage}</ProjectUrl>
+    <RepositoryUrl>${package.repository.url}</RepositoryUrl>
```

Comment:
<Optional>

Since we are now adding this information, do we want to enable source link?
https://github.com/dotnet/sourcelink/blob/master/README.md

Reply:
The problem is that `SourceLink` seems to require that the `git` repository is present (or at least certain files like `.git/HEAD` and `.git/config`, which isn't present on *CodePipeline* executions.

Reply:
This is fine for now, just wanted to raise the possibility.

Reply:
Agreed, just wanting to raise the possibility.